runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp

```diff
@@ -380,19 +381,20 @@ TEST_P(AMDAIENPUDeviceModelParameterizedSixTupleNPU4ColTestFixture,
 
   // TODO(max): maybe there's a way in gtest for the generators to be
   // parameterized?
-  if ((srcStrmSwPortType == CTRL || destStrmSwPortType == CTRL) &&
+  if ((srcStrmSwPortType == StrmSwPortType::CTRL ||
+       destStrmSwPortType == StrmSwPortType::CTRL) &&
       (srcChan > 0 || dstChan > 0))
     return;
-  if (srcStrmSwPortType == TRACE && srcChan > 0) return;
-  if (srcStrmSwPortType == NORTH && srcChan > 3) return;
-  if (destStrmSwPortType == SOUTH && dstChan > 3) return;
+  if (srcStrmSwPortType == StrmSwPortType::TRACE && srcChan > 0) return;
+  if (srcStrmSwPortType == StrmSwPortType::NORTH && srcChan > 3) return;
+  if (destStrmSwPortType == StrmSwPortType::SOUTH && srcChan > 3) return;
```

Comment:
`dstChan` -> `srcChan`, this was a mistake?

Reply:
hmm